clavery63/point-click
# make menus/cursor configurable

This makes the positions of the menu options configurable instead of hard-coded.
It also makes the menu backgrounds and cursor image possible to override

TODO:
- [x] Create admin fields for overriding menu and inventory bg images
- [x] Create an admin widget for positioning the menu options (maybe as a first pass it's just some text fields)
- [x] Enable moving MiniMap and other buttons
- [x] Enable picking background color for viewport and text overlay

How To Test:
1. Visit http://localhost:3000/admin/test-game/config/ui (or equivalent for another game)
2. Try to change the images and color with the available fields
3. Try dragging around menu texts and the minimap
4. Click on a menu verb and change the verb name and default text
5. After changing some stuff, save (need to do in the config screen for now), and try playing the game and confirm the changes are reflected there
